mono/tests/Makefile.am

```diff
@@ -2942,22 +2942,29 @@ EXTRA_DIST += test_lldb.py test-lldb.cs
 test-lldb: test-lldb.exe
 	python test_lldb.py $(JITTEST_PROG)
 
-noinst_LTLIBRARIES = libtest.la
+noinst_LTLIBRARIES = libtest.la libtest-c.la
```

Comment:
This could instead be a regular non-libtool library, and below LIBADD the .o files.